01/app.js

```diff
@@ -0,0 +1,28 @@
+const formElement = document.querySelector('form');
+
+if(formElement){
+    formElement.addEventListener('submit', function(e) {
+        e.preventDefault();
+
+        const firstName = formElement.elements['firstName'];
+        const lastName = formElement.elements['lastName'];
+
+        if(firstName !== '' && lastName !== ''){
```

Comment:
Utaj należy sprawdzić czy wartość nie jest pusta tj. `firstName.value` - inaczej to nie zadziała.